cli/pcluster/cli_commands/delete.py

```diff
@@ -117,5 +120,41 @@ def _delete_cluster(cluster_name, nowait):
         sys.stdout.flush()
         sys.exit(1)
     except KeyboardInterrupt:
+        terminate_compute_fleet = False
         LOGGER.info("\nExiting...")
         sys.exit(0)
+    finally:
+        if terminate_compute_fleet:
```

Comment:
What happens if terminate_compute_fleet is False and there are node running? Will they just be running until their internal daemons shut them down?

Reply:
There is anyway a CustomResource ran as part of the stack deletion that tries to delete all of them